rules/Symfony73/Rector/Class_/CommandHelpToAttributeRector.php

```diff
@@ -0,0 +1,214 @@
+TXT)]
+final class SomeCommand extends Command
+{
+}
+CODE_SAMPLE
+                ),
+            ]
+        );
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [Class_::class];
+    }
+
+    /**
+     * @param Class_ $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        if ($node->isAbstract()) {
+            return null;
+        }
+
+        if (! $this->reflectionProvider->hasClass(SymfonyAttribute::AS_COMMAND)) {
+            return null;
+        }
+
+        if (! $this->isObjectType($node, new ObjectType(SymfonyClass::COMMAND))) {
+            return null;
+        }
+
+        $asCommandAttribute = $this->ensureAsCommandAttribute($node);
+
+        foreach ($asCommandAttribute->args as $arg) {
+            if ($arg->name?->toString() === 'help') {
+                return $node;
+            }
+        }
+
+        $configureClassMethod = $node->getMethod(CommandMethodName::CONFIGURE);
+        if (! $configureClassMethod instanceof ClassMethod) {
+            return null;
+        }
+
+        $helpExpr = $this->findAndRemoveSetHelpExpr($configureClassMethod);
+        if (! $helpExpr instanceof String_) {
+            return null;
+        }
+
+        $wrappedHelp = new String_(
+            $helpExpr->value,
+            [
+                Attributekey::KIND => String_::KIND_NOWDOC,
+                AttributeKey::DOC_LABEL => 'TXT',
+            ]
+        );
+
+        $asCommandAttribute->args[] = new Arg($wrappedHelp, false, false, [], new Identifier('help'));
+
+        if ($configureClassMethod->stmts === []) {
+            unset($configureClassMethod);
+        }
+
+        return $node;
+    }
+
+    private function ensureAsCommandAttribute(Class_ $class): Attribute
+    {
+        foreach ($class->attrGroups as $attrGroup) {
+            foreach ($attrGroup->attrs as $attribute) {
+                if ($this->nodeNameResolver->isName($attribute->name, SymfonyAttribute::AS_COMMAND)) {
+                    return $attribute;
+                }
+            }
+        }
+
+        $attrGroup = $this->phpAttributeGroupFactory->createFromClass(SymfonyAttribute::AS_COMMAND);
+        $class->attrGroups[] = $attrGroup;
+
+        return $attrGroup->attrs[0];
+    }
+
+    /**
+     * Returns the argument passed to setHelp() and removes the MethodCall node.
+     */
+    private function findAndRemoveSetHelpExpr(ClassMethod $configureMethod): ?String_
+    {
+        $helpString = null;
+
+        $this->traverseNodesWithCallable(
+            (array) $configureMethod->stmts,
+            function (Node $node) use (&$helpString): ?Expr {
+                if (! $node instanceof MethodCall) {
+                    return null;
+                }
+
+                if (! $this->isName($node->name, 'setHelp')) {
+                    return null;
+                }
+
+                $argExpr = $node->getArgs()[0]
```

Comment:
ensure `[0]` index exists and not `firstClassCallable()` something like this pseudo code

```php
if ($node->isFirstClassCallable() || ! isset($node->getArgs()[0])) {
   return null;
}
```